concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2777,37 +2778,99 @@ jobs:
           release: pipeline-pool
 
   - name: clear-cloudfoundry-credentials
+    serial: true
     plan:
+    - aggregate:
+      - get: paas-cf
+        passed: ['rotate-cloudfoundry-credentials']
       - get: cf-secrets
+        passed: ['rotate-cloudfoundry-credentials']
+        trigger: true
+
+    - task: clear-passwords
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: governmentpaas/spruce
+            tag: 465642da06051a55630d39c899697b678f66a7f7
+        inputs:
+          - name: paas-cf
+          - name: cf-secrets
+        outputs:
+          - name: modified-cf-secrets
+        run:
+          path: sh
+          args:
+            - -e
+            - -c
+            - |
+              spruce merge \
+                cf-secrets/cf-secrets.yml \
+                paas-cf/concourse/resources/cf-secrets-keep.yml | \
+                spruce merge - paas-cf/concourse/resources/finalize.yml \
+                > modified-cf-secrets/cf-secrets.yml
+      on_success:
+        put: cf-secrets
+        params:
+          file: modified-cf-secrets/cf-secrets.yml
+
+  - name: rotate-cloudfoundry-credentials
```

Comment:
Can we order this job before `clear-cloudfoundry-credentials` so that the order in this file matches the order they run in the pipeline. It took me quite a while to figure out that they run in this order with the pipeline as it is now.

Reply:
Changed.